### PR TITLE
create linux-kernel.rst

### PR DESCRIPTION
**Pull request by rebeccasf**
First draft on kernel development guide with steps to setup tooling framework and show steps to modify linux kernel configuration for development use.

### CODE REVIEW

**Comment by mvincerx** [comment]
@rebeccasf. Thanks for your contribution!  I'd like one of our TMEs in JF Oregon to review this before you proceed further. @bktan8, please see my email.

**Comment by puneetse** [comment]
Thanks @rebeccasf . I actually have a draft on this topic that goes into more detail, which I've been working on since our last conversation. I will upload and share that with you for review soon.

File: source/clear-linux/tutorials/linux-kernel.rst [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+* linux
+
+* linux-kvm
+
+* linux-hyperv
+
+* linux-lts
+
+Search for all kinds of Linux kernel variants supported by |CL| here: `Clear Linux Packages`_
```
Comment (bktan8) [inline source/clear-linux/tutorials/linux-kernel.rst]: "Search for all kinds" is a little awkward.  What does the "Clear Linux Packages" link go to?

**Comment by mvincerx** [comment]
Hi @puneetse, do you want to use this branch, or do you prefer to open your own branch? It's your call.
@bktan8, Thanks for your feedback on this. I'll assure that your comments are addressed in whatever final form this PR takes. 
@rebeccasf.  Thanks for your work. Please hold on development of this branch for now.

**Comment by puneetse** [comment]
@mvincerx I will create a new branch be sure to include everyone here for review.

**Comment by mvincerx** [comment]
Thank you @puneetse !

**Comment by bktan8** [comment]
@mvincerx @puneetse - So which PR should I review?

**Comment by rebeccasf** [comment]
Thanks @puneetse 
I will close this PR now.

**Pull request closed without merging**